packages/plan-form-core/src/formHelpers/helpers.ts

```diff
@@ -726,6 +726,20 @@ export function generatePlanDefinition(
   };
 }
 
+/**
+ * Get date by passing ISOString
+ * returns new date neglecting timezone
+ *
+ * @param {string} e - iso date string
+ * @returns {Date} - get formatted date
+ */
+export function dateFormatter(e: string) {
+  const dateOptions = { timeZone: 'UTC', month: 'long', day: 'numeric', year: 'numeric' };
+  const dateFormat = new Intl.DateTimeFormat('en-US', dateOptions);
```

Comment:
feeling iffy about the hard-cded values especially the localization information.